src/player.cpp

```diff
@@ -0,0 +1,63 @@
+#include "player.h"
+#include <iostream>
+
+// Helper function
+/**
+ * @brief Trim trailing whitespace from a string
+ * 
+ * @param s string to trim
+ */
+void rtrim(std::string& s) {
+  s.erase(s.find_last_not_of(" \n\r\t") + 1);
+} 
+
+Player::Player(bool is_cpu): is_cpu_(is_cpu) {
+  id_ = 0;
+  score_ = 0;
+  last_move_ = 0;
+  board_size_ = 0;
+}
+
+void Player::init(unsigned int id, unsigned int board_size) {
```

Comment:
nit - do we need a constructor AND an init? do we want to call init in the ctor?

Reply:
Good question. I currently create players, then I pick board size. So I would have to address that somehow...

Reply:
fine for now if you think so